net-mgmt/telegraf/src/opnsense/mvc/app/controllers/OPNsense/Telegraf/forms/output.xml

```diff
@@ -35,6 +35,36 @@
         <type>text</type>
         <help>Set the password for authentication.</help>
     </field>
+    <field>
+        <id>output.influx_v2_enable</id>
+        <label>Enable Influx v2 Output</label>
+        <type>checkbox</type>
+        <help>This will enable InfluxDB v2 as output. Format is without square brackets, just like http://192.168.0.1:9999.</help>
+    </field>
+    <field>
+        <id>output.influx_v2_url</id>
+        <label>Influx v2 URL</label>
+        <type>text</type>
+        <help>Set the URL where metrics shoud be sent to.</help>
+    </field>
+    <field>
+        <id>output.influx_v2_token</id>
+        <label>Influx v2 Token</label>
+        <type>text</type>
+        <help>Influx v2 authentication token.</help>
+    </field>
+    <field>
+        <id>output.influx_v2_organization</id>
+        <label>Influx v2 Organization Name</label>
+        <type>text</type>
+        <help>Influx v2 organization name.</help>
+    </field>
+    <field>
+        <id>output.influx_v2_bucket</id>
+        <label>Influx v2 Bucket Name</label>
```

Comment:
You can remove `Name` and make help-text more descriptive

Reply:
should be a bit better now :)